llvm/include/llvm/Transforms/Instrumentation/LowerAllowCheckPass.h

```diff
@@ -20,13 +20,22 @@
 
 namespace llvm {
 
+struct LowerAllowCheckOptions {
```

Comment:
I would rather make it nested class: LowerAllowCheckPass::Options

Reply:
Done